### PR TITLE
doc: Add Opera to list of browsers supporting SW.

### PR DESCRIPTION
**Pull request by nongaap**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
 
         <h3>Browser Support</h3>
         <p>
-          ServiceWorker is currently supported in Chrome &amp; Firefox, and
+          ServiceWorker is currently supported in Opera, Chrome, &amp; Firefox, and
```
Comment (gr2m) [inline public/index.html]: can you add Opera after Firefox, to order the names alphabetically? Thanks!

File: public/6.html [hunk 2]
```diff
@@ -53,14 +53,14 @@
...
   }
 }
 function updateColorOnList(color) {
   $('.colorList').find('[data-id="'+color.id+'"]')
     .text(color.name+' (id: '+color.id+')');
   if (color.id === lastColorId) {
-    $('html').css('background', color.name);
+    $(document.body).css('background', color.name);
   }
 }
```
Comment (gr2m) [inline public/6.html:65]: can you send a separate pull request for these changes, and keep this one to the Opera change only?

**Review by gr2m — CHANGES_REQUESTED** [review]

File: public/6.html (hunk 2 above)
Comment (nongaap) [reply public/6.html:65]: @gr2m I realize I was working off a branch that had these changes in place and previously made a PR for. https://github.com/hoodiehq/hoodie-camp-tutorial/pull/29

**Comment by nongaap** [comment]
@gr2m requested changes made.

**Review by varjmes — APPROVED** [review]

**Review by gr2m — APPROVED** [review]

**Comment by gr2m** [comment]
thanks 👍
